sky/cli.py

```diff
@@ -1673,6 +1673,8 @@ def cost_report(all: bool):  # pylint: disable=redefined-builtin
     on the cloud console.
     """
     cluster_records = core.cost_report()
+    total_cost = status_utils.get_estimated_total_cost_for_all_records(
+        cluster_records)
```

Comment:
```suggestion
    total_cost = sum(record['total_cost'] for record in cluster_records)
```

Principle: Avoid shallow function that is only used once.